src/apps/external_executor/external_executor.cpp

```diff
@@ -35,11 +38,17 @@ namespace externalexecutor
 
   class EndpointRegistry : public ccf::UserEndpointRegistry
   {
-    // Note: As a temporary solution for testing, this app stores a single Tx,
-    // stolen from a StartTx RPC rather than a real client request
-    std::unique_ptr<kv::CommittableTx> active_tx = nullptr;
+    struct PendingRequest
+    {
+      std::unique_ptr<kv::CommittableTx> tx = nullptr;
+      externalexecutor::protobuf::RequestDescription request_description;
+      std::shared_ptr<http::HTTPResponder> responder;
+    };
+    std::queue<PendingRequest> pending_requests;
```

Comment:
I've opted not to rename this one, as I think the fact this is a queue is just an implementation detail. As far as most of the code is concerned it's some kind of container for adding and fetching `PendingRequest`s. The actual type and details may change in future.

Reply:
Sure, makes sense. The only reason I think I suggested a rename here is because elsewhere I suggested `pending_request` and was trying to differentiate this one more explicitly.